packages/back-end/src/routers/demo-datasource-project/demo-datasource-project.controller.ts

```diff
@@ -289,7 +290,9 @@ spacing and headings.`,
       owner: ASSET_OWNER,
       datasource: datasource.id,
       project: project.id,
-      goalMetrics: metrics
+      goalMetrics: metrics.slice(0, 1).map((m) => m.id),
```

Comment:
oh this is actually quite easy to do as you said 😂  -- nice!

Reply:
The first one made sense as the goal, and this was the simplest way to do it.